package.json

```diff
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/Instabug/instabug-reactnative#readme",
   "rnpm": {
     "android": {
-      "packageInstance": "new RNInstabugReactnativePackage.Builder(\"YOUR_ANDROID_APPLICATION_TOKEN\",MainApplication.this)\n.setInvocationEvent(\"shake\")\n.setPrimaryColor(\"#1D82DC\")\n.setFloatingEdge(\"left\")\n.setFloatingButtonOffsetFromTop(250)\n.build()"
+      "packageInstance": "new RNInstabugReactnativePackage.Builder(\"YOUR_ANDROID_APPLICATION_TOKEN\",MainApplication.this)\n                    .setInvocationEvent(\"shake\")\n                    .setPrimaryColor(\"#1D82DC\")\n                    .setFloatingEdge(\"left\")\n                    .setFloatingButtonOffsetFromTop(250)\n                    .build()"
```

Comment:
@salmatarzi Also please don't exceed standard line length which is `80` characters per line.